src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -3641,6 +3647,18 @@ float ViewProviderSketch::getScaleFactor()
     }
 }
 
+void ViewProviderSketch::InitItemsSizes()
+{
+    int defaultFontSize = QApplication::fontMetrics().height();
+    int ldpi = QApplication::desktop()->logicalDpiX();
+    float k = 96./ldpi;
+    float view3D_factor = 1.25;

```

Comment:
Please add some comments. The factors are clear to you for now, but what about others who investigate the code or for you 2 years later? Therefore also meaningful variable names help, so not just the name 'k'. For example i don't know what 96/ldpi since it is not clear to me where the 96 comes from.